install/set_gigabyte_node_bmc_to_factory_defaults.md

```diff
@@ -1,111 +1,106 @@
 # Set Gigabyte Node BMC to Factory Defaults
 
-### Prerequisites
+## Prerequisites
+
+**Note**: This section refers to scripts that exist only in the PIT environment. If necessary, copy the LiveCD data from a different machine to get these scripts.
 
 Use the management scripts and text files to reset Gigabyte BMC to factory default settings. Set the BMC to the factory default settings in the following cases:
 
-- There are problems using the ipmitool command and Redfish does not respond
-- There are problems using the ipmitool command and Redfish is running
-- When BIOS or BMC flash procedures fail using Redfish
+- There are problems using the `ipmitool` command and Redfish does not respond
+- There are problems using the `ipmitool` command and Redfish is running
+- When BIOS or BMC flash procedures fails using Redfish
   - Run the `do_bmc_factory_default.sh` script
-  - Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes seconds
+  - Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes (300 seconds).
 - If booted from the PIT node:
-  - the firmware packages are located in the HFP package provided with the Shasta release
-  - the required scripts are located in `/var/www/fw/river/sh-svr-scripts`
+  - The firmware packages are located in the HFP package provided with the Shasta release
+  - The required scripts are located in `/var/www/fw/river/sh-svr-scripts`
 
-### Procedure
+## Procedure
 
-#### Apply the BMC Factory Command
+### Apply the BMC Factory Command
 
 1. Create a `node.txt` file and add the target node information as shown:
 
-   Example `node.txt` file with two nodes:
+    Example `node.txt` file with two nodes:
 
-   ```screen
+    ```screen
     10.254.1.11 x3000c0s9b0 ncn-w002
     10.254.1.21 x3000c0s27b0 uan01
-   ```
+    ```
 
    Example `node.txt` file with one node:
 
-   ```screen
+    ```screen
     10.254.1.11 x3000c0s9b0 ncn-w002
-   ```
-
-2. Use Redfish to reset the BMC to factory default.
+    ```
 
-   The BMC is running 12.84.01 or a later version.
+2. Use Redfish to reset the BMC to factory default, the BMC is running 12.84.01 or later version, run:
 
-   ```bash
-   ncn-w001# sh do_Redfish_BMC_Factory.sh
-   ```
+    ```bash
+    ncn-w001# sh do_Redfish_BMC_Factory.sh
+    ```
 
    - Alternatively, use `ipmitool` to reset the BMC to factory defaults:
 
-      ```bash
-      ncn-w001# sh do_bmc_factory_default.sh
-      ```
+    ```bash
+    ncn-w001# sh do_bmc_factory_default.sh
+    ```
 
    - Alternatively, use the power control script:
 
-      ```bash
-      ncn-w001# sh do_bmc_power_control.sh raw 0x32 0x66
-      ```
-
-   #### After the BMC has Been Reset to Factory Defaults
+    ```bash
+    ncn-w001# sh do_bmc_power_control.sh raw 0x32 0x66
```

Comment:
Will those always be the two arguments that should be used? Or is the user supposed to change them somehow based on the specific environment?

Reply:
I don't know who is working on Gigabyte firmware updates these days, I'll try Trieu, Tan Kim <tan.trieu@hpe.com>